lib/bundler/cli/outdated.rb

```diff
@@ -91,16 +91,12 @@ def run
           groups = dependency.groups.join(", ")
         end
 
-        outdated_gems_list << { :active_spec => active_spec,
```

Comment:
Is there any reason why you're changing the style?

Reply:
yes duplicated code, when I did the feature group I duplicated the hash https://github.com/bundler/bundler/pull/4841/files#diff-1dacde23b0f908c590b785187df11b00R77

Reply:
I will push something better

Reply:
What @colby-swandale meant is that you shouldn't change the current style. First, because we should first agree on any style changes, and apply them consistently, but mainly because it hides the actual point of your PR, namely, fixing the small issue with output pluralization. So the idea would be to undo these particular changes and leave this part of the code untouched because it's not needed for this fix.

Reply:
I agree.